src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -23,29 +23,10 @@
  */
 class DoctrineChoiceLoader implements ChoiceLoaderInterface
 {
-    /**
-     * @var ChoiceListFactoryInterface
```

Comment:
phpstorm requires those annotations for autocomplete AFAIK

Reply:
Phpstorm doesn't need them when the type is inferable from the constructor.